SerialPrograms/Source/PokemonSV/Programs/Farming/PokemonSV_MaterialFarmerTools.h

```diff
@@ -94,13 +96,36 @@ void lets_go_movement0(BotBaseContext& context);
 
 void lets_go_movement1(BotBaseContext& context);
 
-bool is_sandwich_expired(WallClock last_sandwich_time);
+bool is_sandwich_expired(WallClock last_sandwich_time, int time_per_sandwich);
```

Comment:
Specify what units this is in. Or better yet, change it to an `std::chrono` type.